app/graphql/schema.graphql

```diff
@@ -569,6 +569,62 @@ type Demarche {
   title: String!
 }
 
+"""
+Une demarche
+"""
+type DemarcheDescriptor {
```

Comment:
Je rajouterai bien le commentaire que tu as mis dans la PR ici :

```
# We don't want to expose the full Demarche type on Dossiers, because it would open the door to recursive queries that we want to avoid.
# DemarcheDescriptorType is a lightweight representation of Demarche metadata.
```